### PR TITLE
Sample Sub-flow for SCVOmniFlow Pilot Customers to use in their inbound flows

### PR DESCRIPTION
**Pull request by yesaiarasan**

### CODE REVIEW

File: ContactFlows/Sample_SCV_Inbound_Subflow [hunk 1]
```diff
@@ -0,0 +1 @@
...":[{"name":"FunctionArn","value":"arn:aws:lambda:us-west-2:352672060859:function:Routing-InvokeTelephonyIntegrationApiFunction-X1JDRJ0XEO2H","namespace":null},{"name":"TimeLimit","value":"8"},{"name":"Parameter","key":"methodName","value":"executeOmniFlow","namespace":null},{"name":"Parameter","key":"flowDevName","value":"flowApiName","namespace":"User Defined"},{"name":"Parameter","key":"fallbackQueue","value":"fallbackQueue","namespace":"User Defined"},{"name":"Parameter","key":"flowInput-relatedRecordId","value":"005001002003004","namespace":null},{"name":"Parameter","key":"contactId","value":"contactId","namespace":"User Defined"}],"metadata":{"position":{"x":213,"y":15},"dynamicMetadata":{"methodName":false,"flowDevName":true,"fallbackQueue":true,"flowInput-relatedRecordId":false,"contactId":true},"useDynamic":false},"target":"Lambda"}],"version":"1","type":"contactFlow","start":"7e93d2e8-1a8b-4ffb-92d5-59a1f8f610a0","metadata":{"entryPointPosition":{"x":15,"y":19},"snapToGrid":false,"name":"Sample_SCV_Inbound_Subflow","description":"Sample SCV Inbound Subflow","type":"contactFlow","status":"published","hash":"6b6fe3e7e893916e38888d6bc0092e3155a23e08472f7b40cd066acbb96a4c8b"}}
```
Comment (nielsenius) [inline ContactFlows/Sample_SCV_Inbound_Subflow:1]: Has the name of this flow, "Sample_SCV_Inbound_Subflow", been reviewed by doc?

Comment (yesaiarasan) [reply ContactFlows/Sample_SCV_Inbound_Subflow:1]: Yes the name of the flow and the play prompts are from doc. Here is is for your reference: https://docs.google.com/document/d/17tjQq-4wV6u9J4U_DGEJuj7v-Nb5GDjHKMh5OToled0/edit?ts=60903fa5#

Comment (nielsenius) [reply ContactFlows/Sample_SCV_Inbound_Subflow:1]: Thanks, @yesaiarasan. @sfyz who should update the readme for contact flows to add a description for this flow? I'm wondering if this is something you've been maintaining, or if it should be added as part of this PR.

Comment (sfyz) [reply ContactFlows/Sample_SCV_Inbound_Subflow:1]: @yesaiarasan @nielsenius  I'm happy to update the README if you'd like me to. Also, one PR suggestion is to format this JSON for easier human-readability...

Comment (sfyz) [reply ContactFlows/Sample_SCV_Inbound_Subflow:1]: (Basically, I'm suggesting to add new lines and indentation... Thank you!)

Comment (nielsenius) [reply ContactFlows/Sample_SCV_Inbound_Subflow:1]: Thanks, Yuvi! Yes, I agree on the formatting. It should be easy to use one of the JSON pretty print websites to format it. If you see other contact flow examples they're formatted.

Comment (yesaiarasan) [reply ContactFlows/Sample_SCV_Inbound_Subflow]: Good point. I have updated with formatting.

Comment (sfyz) [reply ContactFlows/Sample_SCV_Inbound_Subflow]: Looks great. Thanks!

**Review by nielsenius — APPROVED** [review]

**Review by sfyz — APPROVED** [review]
👍

File: ContactFlows/Sample_SCV_Inbound_Subflow (hunk 1 above)
Comment (sfyz) [reply ContactFlows/Sample_SCV_Inbound_Subflow]: Great! Merging now...

**Pull request merged**